dashboard/load_dashboard.py

```diff
@@ -27,3 +32,15 @@
     st.altair_chart(temp_bar_chart, use_container_width=True)
     st.altair_chart(temp_line_chart, use_container_width=True)
     st.altair_chart(moist_bar_chart, use_container_width=True)
+
+    # LONG TERM VISUALISATIONS:
+
+    longterm_df = concat_csvs(extract_s3_files())
+
+    water_count_bar = get_times_watered(longterm_df)
+    max_temp_bar = get_max_temp_per_plant(longterm_df)
```

Comment:
Very neat dashboarding code. For a future improvement we could consider using Altair's columns to divide it.